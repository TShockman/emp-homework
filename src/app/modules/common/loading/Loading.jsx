import React, {PureComponent} from 'react';
import style from './loading.scss';

export default class Loading extends PureComponent {
  render() {
    return <div className="loading"><div className="loader"/></div>;
  }
};