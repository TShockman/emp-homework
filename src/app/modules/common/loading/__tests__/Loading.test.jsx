import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Loading from '../';

const renderer = new ReactShallowRenderer();

describe('Loading', () => {
  it('Renders consistently', () => {
    expect(renderer.render(<Loading/>)).toMatchSnapshot();
  });
});