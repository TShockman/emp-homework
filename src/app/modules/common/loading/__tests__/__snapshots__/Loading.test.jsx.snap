// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Renders consistently 1`] = `
<div
  className="loading"
>
  <div
    className="loader"
  />
</div>
`;
